sources/editor/Stride.Assets.Presentation/Stride.Assets.Presentation.csproj

```diff
@@ -15,21 +15,15 @@
     <EnableDefaultPageItems>false</EnableDefaultPageItems>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.Build" Version="16.11.0" ExcludeAssets="runtime" />
-    <PackageReference Include="AvalonEdit" Version="6.0.0" />
+    <PackageReference Include="Microsoft.Build" Version="17.0.0" ExcludeAssets="runtime" />
+    <PackageReference Include="AvalonEdit" Version="6.1.3.50" />
     <PackageReference Include="System.Reactive.Linq" Version="5.0.0" />
     <PackageReference Include="System.Reactive" Version="5.0.0" />
-    <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Features" Version="3.4.0" />
-    <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Scripting" Version="3.4.0" />
-    <Reference Include="RoslynPad.Editor.Windows">
-      <HintPath>$(StrideCommonDependenciesDir)RoslynPad\netcoreapp3.1\RoslynPad.Editor.Windows.dll</HintPath>
-    </Reference>
-    <Reference Include="RoslynPad.Roslyn.Windows">
-      <HintPath>$(StrideCommonDependenciesDir)RoslynPad\netcoreapp3.1\RoslynPad.Roslyn.Windows.dll</HintPath>
-    </Reference>
-    <Reference Include="RoslynPad.Roslyn">
-      <HintPath>$(StrideCommonDependenciesDir)RoslynPad\netcoreapp3.1\RoslynPad.Roslyn.dll</HintPath>
-    </Reference>
+    <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Features" Version="3.6.0" />
+    <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Scripting" Version="3.6.0" />
+    <PackageReference Include="RoslynPad.Editor.Windows" Version="1.2.0" />
+    <PackageReference Include="RoslynPad.Roslyn.Windows" Version="3.6.0" />
+    <PackageReference Include="RoslynPad.Roslyn" Version="3.6.0" />
```

Comment:
Removed our roslynpad as it looks like we branched just for the highlight color which is now part of the nuget package